accountKit.go

```diff
@@ -42,9 +45,13 @@ func NewAccountKit(appID, appSecret string) *AccountKit {
 	}
 }
 
-func (a *AccountKit) VerifyByToken(token string) (*Profile, error) {
+func (a *AccountKit) VerifyByToken(userAccessToken string, accountKitAppSecret string) (*Profile, error) {
```

Comment:
thanks for submitting pr.

just one quick question: what is difference between this `accountKitAppSecret` and the one existing `AppSecret`?